lib/TerminalDisplay.h

```diff
@@ -526,6 +528,12 @@ public slots:
      */
     void setForegroundColor(const QColor& color);
 
+    /**
+    * hide the mouse cursor after @param delay milliseconds of inactivity
+    * @param delay < 0 deactivates the behavior
+    */
+    void autoHideMouseAfter(int delay);
```

Comment:
Any reason to make it a slot (it's put under `public slots:`)?

Reply:
The idea was for consumers (qterminal) to easily bind this to eg. QSpinBox signals etc. - there's not internal requirement for the slot.

Reply:
OK.